functions/agencies/src/actions/update.ts

```diff
@@ -22,6 +22,11 @@ export const updateAgency = async (request: AgenciesRequest, environment: Stage)
   }
 
   const updateItemCommand = new UpdateCommand({
+    ExpressionAttributeNames: {
+      "#n": "name",
+      "#a": "address",
+      "#p": "phone"
+    },
     ExpressionAttributeValues: {
       ":address": { S: request.address },
```

Comment:
wondering if you need to change these as well to not include the 'S' and just be ` ":address": request.address,